samples/deploy/crd/v1beta1/ApisixUpstream.yaml

```diff
@@ -32,3 +32,342 @@ spec:
+                    - header
+                    - cookie
+                    - consumer
+                key:
+                  type: string
+              required:
+                - type
+            scheme:
+              type: string
+              enum:
+                - http
+                - grpc
+            retries:
+              type: integer
+              minimum: 0
+            timeout:
+              type: object
+              properties:
+                connect:
+                  type: string
+                read:
+                  type: string
+                send:
+                  type: string
+            healthCheck:
+              type: object
+              anyOf:
+                - required:
+                    - active
+                - required:
+                    - active
+                    - passive
+              properties:
+                active:
+                  type: object
+                  properties:
+                    type:
+                      type: string
+                      enum:
+                        - http
+                        - https
+                        - tcp
+                    timeout:
+                      type: number
+                      minimum: 0
+                    concurrency:
+                      type: integer
+                      minimum: 1
+                    host:
+                      type: string
+                      pattern: "^\\*?[0-9a-zA-Z-._]+$"
+                    port:
+                      type: integer
+                      minimum: 1
+                      maximum: 65535
+                    httpPath:
+                      type: string
+                      minLength: 1
+                    strictTLS:
+                      type: boolean
+                    requestHeaders:
+                      type: array
+                      minItems: 1
+                      items:
+                        type: string
+                    healthy:
+                      type: object
+                      properties:
+                        interval:
+                          type: string
+                        httpCodes:
+                          type: array
+                          minItems: 1
+                          items:
+                            type: integer
+                            minimum: 200
+                            maximum: 599
+                        successes:
+                          type: integer
+                          minimum: 1
+                          maximum: 254
+                    unhealthy:
+                      type: object
+                      properties:
+                        interval:
+                          type: string
+                        httpCodes:
+                          type: array
+                          minItems: 1
+                          items:
+                            type: integer
+                            minimum: 200
```

Comment:
According to [RFC 2616](https://tools.ietf.org/html/rfc2616#section-10) http status code can include 1xx.

Should we just follow the norms instead of making decisions for users? There may be some custom status codes.

Reply:
That's limited by APISIX itself, as APISIX limiting the minimum status code to `200`. We may change it after APISIX changing.

Reply:
ok